### ISSUE TITLE
Compatible with bagisto master

### ISSUE THREAD

**Issue by ghermans** [bagisto/laravel-data-faker#24]
Optimize the package to be compatible with the current master branch of Bagisto.

**Comment by ghermans** [comment]
@devansh-webkul & @naresh-webkul while installing the package i get the following output

```
vagrant@homestead:~/bagisto$ composer require bagisto/laravel-datafaker
Using version dev-master for bagisto/laravel-datafaker
./composer.json has been updated
Running composer update bagisto/laravel-datafaker
Loading composer repositories with package information
Updating dependencies
Lock file operations: 3 installs, 0 updates, 0 removals
  - Locking bagisto/laravel-datafaker (dev-master 735feee)
  - Locking laravel/helpers (v1.5.0)
  - Locking mbezhanov/faker-provider-collection (2.0.1)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 3 installs, 0 updates, 0 removals
  - Downloading laravel/helpers (v1.5.0)
  - Syncing bagisto/laravel-datafaker (dev-master 735feee) into cache
  - Installing mbezhanov/faker-provider-collection (2.0.1): Extracting archive
  - Installing laravel/helpers (v1.5.0): Extracting archive
  - Installing bagisto/laravel-datafaker (dev-master 735feee): Cloning 735feee086 from cache
Package swiftmailer/swiftmailer is abandoned, you should avoid using it. Use symfony/mailer instead.
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover

In Container.php line 879:
                                                                       
  Target class [Illuminate\Database\Eloquent\Factory] does not exist.  
                                                                       

In Container.php line 877:
                                                               
  Class "Illuminate\Database\Eloquent\Factory" does not exist  
                                                               

Script @php artisan package:discover handling the post-autoload-dump event returned with error code 1
```

**Comment by naresh-webkul** [comment]
Hi @ghermans ,
Which version of bagisto you are using ?
checked from my end not getting this issue with v1.3.3.

**Comment by ghermans** [comment]
@naresh-webkul i tested with the master branch of bagisto

**Comment by naresh-webkul** [comment]
please test with the stable branch v1.3.3.

**Comment by naresh-webkul** [comment]
If still getting any issue please let me know.

**Comment by naresh-webkul** [comment]
As it is compatible with the latest stable branch that is v1.3.3 . so you may  get some issues with master branch.

**Comment by navneetbhardwaj** [comment]
@ghermans 
Due to the many factories are removed from larave8. that's why showing this issue. 
I used the legacy factories and its worked.
```
composer require laravel/legacy-factories
```

**Comment by jitendra-webkul** [comment]
Fixed in #27 (Brand new faker)
